packages/turf-constrained-tin/test.js

```diff
@@ -0,0 +1,75 @@
+const fs = require('fs');
+const path = require('path');
+const test = require('tape');
+const constrainedTin = require('./');
+
+const points = require(path.join(__dirname,  'test', 'Points.json'));
+const points_grid = require(path.join(__dirname,  'test', 'Points_grid.json'));
+const mercator = require(path.join(__dirname,  'test', 'mercator.json'));
+
+test('tin - z property', t => {
+    const expected = require(path.join(__dirname,  'test', 'Tin.json'));
+    const tinned = constrainedTin(points, [[2, 12]], 'elevation');
+    t.equal(tinned.features[0].geometry.type, 'Polygon');
+    t.equal(tinned.features.length, 24);
+    t.deepEqual(tinned, expected, 'tinned polygons match');
```

Comment:
If you are comparing the result with a specific output you don't really need to check if the result's geometry and length, that's implicit in the deep comparison, isn't it?

Reply:
OK you are right.

Reply:
Removed